grammars/less.cson

```diff
@@ -453,6 +459,161 @@
         'include': '$self'
       }
     ]
+
   'less_builtin_functions':
     'match': '\\b(abs|acos|alpha|argb|asin|atan|average|blue|calc|ceil|color|contrast|convert|convert|cos|darken|data-uri|desaturate|difference|e|escape|exclusion|extract|fade|fadein|fadeout|floor|format|green|greyscale|hardlight|hsl|hsla|hsv|hsva|hsvhue|hsvsaturation|hsvvalue|hue|length|lighten|lightness|luma|max|min|mix|mod|multiply|negation|overlay|percentage|pi|pow|red|replace|round|saturate|saturation|screen|sin|softlight|spin|sqrt|tan|unit)\\b'
     'name': 'support.function.any-method.builtin.less'
+
+  'function_type_checking':
+    'begin': '\\b(isurl|isstring|isnumber|iskeyword|iscolor)(\\()'
+    'beginCaptures':
+      '1':
+        'name': 'support.function.type-checking.less'
+      '2':
+        'name': 'meta.brace.round.css'
+    'end': '\\)'
+    'endCaptures':
+      '0':
+        'name': 'meta.brace.round.css'
+    'patterns': [
+      {
+        'include': '#property_values'
+      }
+      {
+        'include': '#variables'
+      }
+    ]
+
+  'function_default':
+    'match': '\\b(default)(\\()(\\))'
+    'captures':
+      '1':
+        'name': 'support.function.default.less'
+      '2':
+        'name': 'meta.brace.round.css'
+      '3':
+        'name': 'meta.brace.round.css'
+
+  'function_unit_checking':
+    'begin': '\\b(isunit|ispixel|ispercentage|isem)(\\()'
+    'beginCaptures':
+      '1':
+        'name': 'support.function.unit-checking.less'
+      '2':
+        'name': 'meta.brace.round.css'
+    'end': '\\)'
+    'endCaptures':
+      '0':
+        'name': 'meta.brace.round.css'
+    'patterns': [
+      {
+        'include': '#property_values'
+      }
+      {
+        'include': '#variables'
+      }
+    ]
+
+  'mixins':
+    'name': 'meta.mixin.less'
+    'begin': '(\\.[_a-zA-Z][a-zA-Z0-9_-]*)(?= *\\()'
```

Comment:
What do you think about giving the `.` its own punctuation scope?  Also, please use `\\s*` instead of `<space>*` for extra clarity.

Reply:
Re: giving `.` it's own scope. Good idea. What would that be called?

Re: using `\\s*` vs ` *`, I agree that's clearer, but the issue is that the space must be horizontal only. Is there a PCRE horizontal whitespace option? IIRC, I think that might be `\\h`?

Reply:
You're correct that `\\s` includes vertical whitespace (like `\n` and `\v`), but in practice it doesn't really matter because matches are restricted to single lines only.

I would think that `.` should be scoped as `punctuation.definition.mixin.less`.

Reply:
I see.. Good to know.

Re: `punctuation.definition.mixin.less` -- Sounds good to me.